packages/nameguard-react/src/components/ReportIcon/index.tsx

```diff
@@ -8,11 +8,11 @@ import {
 import cc from "classcat";
 
 import { Tooltip } from "../Tooltip/Tooltip";
-import { RatingLoadingIcon } from "../icons/RatingLoadingIcon";
 import { RatingIcon, RatingIconSize } from "../Report/RatingIcon";
 import { checkResultCodeTextColor, ratingTextColor } from "../../utils/text";
 import { ENSName } from "@namehash/ens-utils";
 import { UnknownReportIcon } from "../UnknownReportIcon/UnknownReportIcon";
+import { LoadingReportIcon } from "../LoadingReportIcon/LoadingReportIcon";
 
 type ReportShieldProps = {
   onClickOverride?: (ensName: ENSName) => void;
```

Comment:
GitHub won't let me leave this comment where I want to. This comment is actually for line 60 below where we generate the URL for a NameGuard report for an `ENSName`.

Please create a utility function for this inside the NameGuard SDK package. It should take in any `ENSName` and return a URL to inspect that name.

Then please look for opportunities to refactor existing code to call into that new utility function. Thanks.